src/main/webapp/app/iris/websocket.service.ts

```diff
@@ -23,6 +24,9 @@ export class IrisWebsocketService implements OnDestroy {
         this.accountService.identity().then((user: User) => {
             this.user = user!;
         });
+        this.connectionStateSub = this.jhiWebsocketService.connectionState.subscribe((newState) => {
```

Comment:
This will lead `websocket.service.spec` failed, so complete this part or remove it.